eva_submission/eload_submission.py

```diff
@@ -74,6 +74,11 @@ def _get_dir(self, key):
     def now(self):
         return datetime.now()
 
+    def _unique_analysis_alias(self, analysis_alias):
```

Comment:
Can this just be a generic `_unique_alias` method?  Then you can reuse it to create the project alias as well.

Reply:
I guess the majority of usage is for analysis alias, so changing it might be weird... up to you.

Reply:
No it makes more sense to apply it to project and analysis. I've changed it.